src/main/java/org/terasology/inGameHelp/systems/InGameHelpClientSystem.java

```diff
@@ -58,6 +70,11 @@ public void onInGameHelpButton(InGameHelpButton event, EntityRef entity) {
         }
     }
 
+	/**
+	 * Show the help screen for the document
```

Comment:
And off by a few spaces again! Autoformat in IDEA to fix indentation errors like these.